api/tests/test_block_processor.py

```diff
@@ -22,25 +22,31 @@ async def main():
+        RetrievedTokenTransfer(transactionHash='0x3093bb2c8cbf1d229594d2cb8b172558fb6730721d9ef561b6400543a59f3734', registryAddress='0x049aba7510f45BA5b64ea9E658E342F904DB358D', fromAddress='0x0000000000000000000000000000000000000000', toAddress='0x54A5A62D34d3C55Cb35Bb8B4d4C958c820c5f322', operatorAddress='0x54A5A62D34d3C55Cb35Bb8B4d4C958c820c5f322', tokenId='111875105896522767201998009571114989808106447240989160672650322275489228602620', amount=1, value=0, gasLimit=147908, gasPrice=40000000000, gasUsed=115697, blockNumber=12839305, blockHash='0x7e36a9f127f3413595ef0f1c4165e4547d20ab566b66fc51a3cbe9eb048f3154', blockDate=datetime.datetime(2021, 7, 16, 18, 41, 41), tokenType='erc721'), 
+        RetrievedTokenTransfer(transactionHash='0x7d15427902a0ae5cb45b20cb38ab4ef0aab204ba1e4ae120f2b3b4d1c40cb688', registryAddress='0x395E5461693e0bB5EC78302605030050f69e628d', fromAddress='0x913C7fa57e6690f96B4AEB65553f0ED3664CAF8b', toAddress='0xA8E3582964ffA96621a767468b1D66207211df00', operatorAddress='0x913C7fa57e6690f96B4AEB65553f0ED3664CAF8b', tokenId='1861', amount=1, value=0, gasLimit=97681, gasPrice=40000000000, gasUsed=82681, blockNumber=12839305, blockHash='0x7e36a9f127f3413595ef0f1c4165e4547d20ab566b66fc51a3cbe9eb048f3154', blockDate=datetime.datetime(2021, 7, 16, 18, 41, 41), tokenType='erc721'), 
+        RetrievedTokenTransfer(transactionHash='0x610b0e54f476ecada3fcc5afe330e967b0f8cc22dafa4cd1983f0f2a204bead3', registryAddress='0xd07dc4262BCDbf85190C01c996b4C06a461d2430', fromAddress='0xe46ec6498B186AaA8C2222c703c463982734D01A', toAddress='0x7AFB8fAC50362DDf276df66a8010ff032BBE1345', operatorAddress='0x4feE7B061C97C9c496b01DbcE9CDb10c02f0a0Be', tokenId='640491', amount=18, value=1826550000000000000, gasLimit=329641, gasPrice=40000000000, gasUsed=325169, blockNumber=12839305, blockHash='0x7e36a9f127f3413595ef0f1c4165e4547d20ab566b66fc51a3cbe9eb048f3154', blockDate=datetime.datetime(2021, 7, 16, 18, 41, 41), tokenType='erc1155single'),
+        RetrievedTokenTransfer(transactionHash='0x9e52523fc2de27af5e927d20a5b895024c46d7e9cebe758303c9768c3f756c19', registryAddress='0x495f947276749Ce646f68AC8c248420045cb7b5e', fromAddress='0xDc62e941fDDBDdDFc666B133E24E0C1aFae11847', toAddress='0xC4e245226537A0B1c5AF30079Eb0251504b42793', operatorAddress='0x750a32FAB9AE3B4ba3c15fF3d140b19aC92f25D8', tokenId='99683587586696680377209091673774721318238762396163454333212325529815842029569', amount=1, value=50000000000000000, gasLimit=228450, gasPrice=40000000000, gasUsed=218447, blockNumber=12839305, blockHash='0x7e36a9f127f3413595ef0f1c4165e4547d20ab566b66fc51a3cbe9eb048f3154', blockDate=datetime.datetime(2021, 7, 16, 18, 41, 41), tokenType='erc1155single')
     ]
-    assert result == expected
+    assert (result == expected)
```

Comment:
could you please add a block with a erc1155batch? not sure how you can find one but would be great to see